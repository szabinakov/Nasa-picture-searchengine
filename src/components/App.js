import React, {useState} from 'react';
import '../styles/App.css';

import Search from './Search'
import SearchResult from './SearchResult';

function App() {
  const [searchResult, setSearchResult] = useState([])

  return (
    <>
    <div className="App">
      <img 
      className='nasaLogo'
      src=' https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg' 
      alt='nasalogo'/>
      <Search setSearchResult={setSearchResult}/>
    </div>
     <SearchResult results={searchResult}/>
    </>
  );
}

export default App;
