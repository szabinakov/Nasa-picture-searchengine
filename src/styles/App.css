.App {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.nasaLogo{
    width: 600px;
}