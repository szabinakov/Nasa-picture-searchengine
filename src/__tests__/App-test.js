import React from 'react';
import { render } from '@testing-library/react';
import App from '../components/App.js';

describe('App', () => {
    const {asFragment} = render (<App/>)

    it('it renders correctly', () => {
        
        expect(asFragment()).toMatchSnapshot()
    })
})