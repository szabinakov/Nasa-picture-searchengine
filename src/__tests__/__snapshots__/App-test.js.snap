// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App it renders correctly 1`] = `
<DocumentFragment>
  <nav>
    <ul>
      <a
        href="/"
      >
        <li
          class="Link"
        >
          Home
        </li>
      </a>
      <a
        href="/about"
      >
        <li
          class="Link"
        >
          About
        </li>
      </a>
    </ul>
  </nav>
  <div
    class="Home"
  >
    <img
      alt="nasalogo"
      class="nasaLogo"
      src=" https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <form
      class="searchForm"
    >
      <input
        class="searchInput"
        type="text"
      />
      <button
        class="searchButton"
        type="submit"
      >
        Search!
      </button>
    </form>
  </div>
  <p>
    No Results!
  </p>
</DocumentFragment>
`;
